index.bs

```diff
@@ -1076,9 +1081,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Server-side-resident Public Key Credential Source</dfn>
 : <dfn>Non-Resident Credential</dfn>
-:: A [=Server-side-resident Public Key Credential Source=], or [=Non-Resident Credential=] for short, is a [=public key credential
-    source=] whose [=credential private key=] is not stored in the [=authenticator=], [=client=] or [=client device=], but is instead
-    encrypted (i.e. wrapped) and returned to the server as the [=credential ID=]. See also: [=server-side credential storage modality=].
+:: A [=Server-side-resident Public Key Credential Source=], or [=Non-Resident Credential=] for short,
+    is a [=public key credential source=] that cannot be used in an [=authentication ceremony=]
+    without providing the [=authenticator=] with the [=credential ID=], e.g.,
+    it can only be used when specified in the {{PublicKeyCredentialRequestOptions/allowCredentials}} argument.
+
+    As a consequence, [=client-side=] storage of the [=credential private key=] is not required for a [=non-resident credential=].
```

Comment:
should "[=credential private key=]" here be "[=public key credential source=]" ?